clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -291,55 +381,33 @@ bool CheckedRegionFinder::isUncheckedStruct(QualType Qt, std::set<std::string> &
   }
 }
 
+
 void CheckedRegionFinder::addUncheckedAnnotation(CompoundStmt *S, int Localwild) {
```

Comment:
Bad name? Since this is just the "finder" you are not actually adding any annotations, right? That happens in a subsequent pass.